sdk/python/ragflow_sdk/modules/document.py

```diff
@@ -69,32 +148,32 @@ def download(self):
             response = res.json()
             actual_keys = set(response.keys())
             if actual_keys == error_keys:
-                raise Exception(res.get("message"))
+                raise Exception(response.get("message"))
```

Comment:
Noticed a bug here and fixed it while I was at it. Mentioned the bug here:
https://github.com/infiniflow/ragflow/commit/f007c1c772788326dd4f10b5eec73c51c1ec965e#r170921731